modules/cnf-creating-custom-templates.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+// Epic CNF-9657 (TELCODOCS-1540) (4.16)
+// * edge_computing/cnf-understanding-siteconfig-operator.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cnf-creating-custom-templates_{context}"]
+= Creating custom templates with the {sco}
+
+Create user-defined templates with the {sco}.
```

Comment:
I wonder if we could add this to the "understanding templates" section as an example?